### DISCUSSION TITLE
Multi-domain feasible with plugin ?

### DISCUSSION THREAD

**Discussion by ramshorst** [Elderjs/elderjs#157] [Ideas] [answered]
I was wondering if it seams doable to build a plugin to support multiple domains.
My use case: Generate multiple websites with different domains and the same structure but different data.

**Comment by nickreese** [comment, marked as answer]
@ramshorst Totally doable. Basically Elder.js just outputs pages into a `public` folder. You could use the same routes to generate different languages with different languages. @kiuKisas has a translation plugin which may also help.

**Reply by kinooyume** [reply]
Yeap ! You probably even can use my plugin (https://github.com/kiuKisas/elderjs-plugin-i18n) for that by deactivating everything related to languages. Same sites generated with different data into different folders. Even if I'm not sure it's a good idea for semantic reason though, you should probably take inspiration from that and do you own implementation :).
